packages/db-model/src/model.ts

```diff
@@ -74,84 +75,123 @@ export function createBaseModel(resolver?: ConnectionResolver) {
     /**
      * Generic query builder.
      *
-     * @param {(qb: Knex | Transaction) => QueryBuilder} callback
      * @param {Transaction} [trx]
-     * @returns {Promise<T[]>}
+     * @returns {(Knex.Transaction | Knex)}
      */
-    public static async buildQuery<T>(
-      callback: (qb: Knex | Transaction) => QueryBuilder,
-      trx?: Transaction
-    ): Promise<T[]> {
-      const qb = db.queryBuilder(this.getConnection(), trx);
-      const result = await callback(qb);
-
-      return object.toCamelCase<T[]>(result);
+    public static queryBuilder(trx?: Transaction): Knex.Transaction | Knex {
+      return db.queryBuilder(this.getConnection(), trx);
     }
```

Comment:
Let's remove this method in favor of the above `buildQuery` it makes complex query building cleaner.